sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_region.c

```diff
@@ -145,4 +146,13 @@ bool gnrc_lorawan_validate_dr(uint8_t dr)
     return false;
 }
 
+bool gnrc_lorawan_validate_tx_power(uint16_t tx_power)
```

Comment:
shouldn't `uint8_t` be enough?

Reply:
`NETOPT_TX_POWER` needs `int16_t`. It will need to be corrected to `int16_t`, I suppose.

Reply:
this is not related to the NETOPTs at all. `NETOPT_TX_POWER` is technology independent. It's the task of the netif implementation to convert between `int16_t` and whatever is needed in the standard. So this should be `uint8_t`